packages/server/src/services/user.service.ts

```diff
@@ -72,6 +72,6 @@ export class UserService {
      * @param displayName - Display name to check exists
      */
     public async displayNameExists(displayName: string): Promise<boolean> {
-        return (await this._userRepository.find({ where: { displayName } })) !== undefined;
```

Comment:
I think using findOne instead of find (since we are only ever looking for 1 entry in this context) might be more in-line with the design of the ORM.